scripts/install/production/roles/tgt/tasks/main.yml

```diff
@@ -0,0 +1,104 @@
+---
+#Install tgt and its necessary dependencies
+
+- name: Install packages for tgt CentOS installation
+  package: name={{ item }} state=latest
+  with_items:
+   - gcc
+   - cpan
+   - make
+  become: true
+  when: ansible_distribution == 'CentOS' or ansible_distribution == 'Red Hat Enterprise Linux'
+
+- name: Change SELinux to permissive for CentOS
```

Comment:
This wouldn't be permanent. Do we only want to set this during the installation? to make it permanent we need to edit some file and save it.

Reply:
From my testing, when using the selinux task in ansible it actually does change the file and it persists across reboots.